docs/guides/expectations/creating_custom_expectations/how_to_create_custom_set_based_column_map_expectations.md

```diff
@@ -0,0 +1,268 @@
+
+* `data`: defines the input data of the example as a table/data frame. In this example the table has columns named `lowercase_solfege_scale`, `uppdecase_solfege_scale`, and `mixed`. All of these columns have 8 rows. (Note: if you define multiple columns, make sure that they have the same number of rows.)
+* `tests`: a list of test cases to validate against the data frame defined in the corresponding `data`.
+	* `title` should be a descriptive name for the test case. Make sure to have no spaces.
+	* `include_in_gallery`: This must be set to `True` if you want this test case to be visible in the Gallery as an example.
+	* `in` contains exactly the parameters that you want to pass in to the Expectation. `"in": {"column": "mixed", "mostly": .1}` in the example above is equivalent to `expect_column_values_to_be_in_solfege_scale_set(column="mixed", mostly=0.1)`
+	* `out` is based on the Validation Result returned when executing the Expectation.
+	* `exact_match_out`: if you set `exact_match_out=False`, then you don’t need to include all the elements of the Validation Result object - only the ones that are important to test.
+
+
+If you run your Expectation file again, you won't see any new checkmarks, as the logic for your Custom Expectation hasn't been implemented yet. 
+However, you should see that the tests you've written are now being caught and reported in your checklist:
+
+```
+$ python expect_column_values_to_be_in_solfege_scale_set.py
+
+Completeness checklist for ExpectColumnValuesToBeInSolfegeScaleSet:
+  ✔ Has a library_metadata object
+  ✔ Has a docstring, including a one-line short description
+...
+	Has core logic that passes tests for all applicable Execution Engines and SQL dialects
+		  Only 0 / 2 tests for pandas are passing
+		  Only 0 / 2 tests for spark are passing
+		  Only 0 / 2 tests for sqlite are passing
+		  Failing: basic_positive_test, basic_negative_test
+...
+```
+
+:::note
+For more information on tests and example cases, <br/>
+see our guide on [how to create example cases for a Custom Expectation](../features_custom_expectations/how_to_add_example_cases_for_an_expectation.md).
+:::
+
+### 6. Define your set and connect it to your Expectation
+
+This is the stage where you implement the actual business logic for your Expectation.   
+
+In the case of your Custom `SetBasedColumnMapExpectation`, Great Expectations will handle the actual validation of your data against you set. 
+
+To do this, we replace these:
+
+```python file=../../../../examples/expectations/set_based_column_map_expectation_template.py#L18-L21
+```
+
+with something like this:
+
+```python file=../../../../tests/integration/docusaurus/expectations/creating_custom_expectations/expect_column_values_to_be_in_solfege_scale_set.py#L10-L34
+```
+
+For more detail when rendering your Custom Expectation, you can optionally specify the plural form of a Semantic Type you're validating. 
```

Comment:
Is there a reason we need to specify both the `set_snake_name` and `set_camel_name`? Couldn't we translate from one to the other [e.g. using this](https://github.com/great-expectations/great_expectations/blob/29c6362ad287d1420633560246cb11d163872bbf/great_expectations/util.py#L128)?

Could you also provide some more explanation about the plural form? How does this align with the `set_semantic_name`?

Reply:
To point one: I don't see why not! 🙇 

To point two: Mis-copy on my part. This should be updated to reference set semantic name.

Reply:
❤️

Reply:
Updated re: above